src/liblsquic/lsquic_crt_compress.c

```diff
@@ -440,7 +445,7 @@ int decompress_certs(const unsigned char *in, const unsigned char *in_end,
 {
     int ret;
     size_t i;
-    uint8_t* uncompressed_data, *uncompressed_data_buf;
+    uint8_t* uncompressed_data = NULL, *uncompressed_data_buf = NULL;
```

Comment:
The one that matters is uncompressed_data.  There is a path where it's not initialized.  And when it is initialized through malloc, the return value is not checked.